### PR TITLE
Release request

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

File: xml/obs_request_and_review_system.xml [hunk 1]
```diff
@@ -98,6 +98,12 @@
     <para> can be used to update the devel package information in the target.
     </para>
    </sect3>
+   <sect3>
+    <title><literal>release</literal></title>
+    <para> Is used to release a ready a build. Sources and binaries are copied
```
Comment (tomschr) [inline xml/obs_request_and_review_system.xml:103]: The first sentence sounds strange. :wink:  Do you mean "Is used to release ready/complete/finished build"?

Comment (adrianschroeter) [reply xml/obs_request_and_review_system.xml]: yeah, clarified that

File: xml/obs_request_and_review_system.xml [hunk 2]
```diff
@@ -101,25 +108,22 @@
    <sect3>
     <title><literal>maintenance_incident</literal></title>
     <para> Official request to open a maintenance incident for official support
-     products. This create by developers who want to start an official
-     maintenance process. Find detail in the maintenance chapter about this. If
-     accepted, a new maintenance incident project is created and package
-     sources listed are copied there. All sources of all actions in one request
-     will be merged into the same maintenance incident. </para>
+     products. These requests are created by developers who want to start an official
+     maintenance process. Details are describe in the maintenance chapter. A new
```
Comment (sknorr) [inline xml/obs_request_and_review_system.xml:112]:
```suggestion
     maintenance process. Details are described in the maintenance chapter. A new
```

File: xml/obs_request_and_review_system.xml [hunk 3]
```diff
@@ -101,25 +108,22 @@
...
+     when accepting it. All sources of all actions in one request
+     will be merged into the same maintenance incident project. </para>
    </sect3>
    <sect3>
     <title><literal>maintenance_release</literal></title>
-    <para> Is used to release a ready maintenance update. Unlike
-     maintenance_incident or submit, sources and binaries are copied without a
-     rebuild. Details can be found in maintenance chapter. </para>
+    <para> Is used to release a maintenance update. Unlike sources and binaries are
+     copied without a rebuild. A unique update identifier is also created.
```
Comment (sknorr) [inline xml/obs_request_and_review_system.xml:120]: Either you need to leave in the explanation about maintenance_incident / submit events or the "Unlike" needs to go:

```suggestion
    <para> Is used to release a maintenance update. Sources and binaries are
     copied without a rebuild. &obs; also creates a unique update identifier.
```

**Pull request merged**